packages/sdk/src/_tests/bundle.test.ts

```diff
@@ -0,0 +1,174 @@
+import * as es from "@linear/sdk";
+import { LinearErrorType } from "../types";
+import { startClient, stopClient } from "./test-client";
+
+const bundles = {
+  // umd: require("../../dist/index-umd.min.js"),
+  cjs: require("../../dist/index-cjs.min.js"),
+  es: es,
```

Comment:
test the different bundles

umd testing requires a different jest env which hasn't been prioritised yet